docs/core/tools/dotnet.md

```diff
@@ -5,212 +5,157 @@ ms.date: 02/13/2020
 
----
-
-## Description
-
-`dotnet` is a tool for managing .NET source code and binaries. It exposes commands that perform specific tasks, such as [`dotnet build`](dotnet-build.md) and [`dotnet run`](dotnet-run.md). Each command defines its own arguments. Type `--help` after each command to access brief help documentation.
-
-`dotnet` can be used to run applications, by specifying an application DLL, such as `dotnet myapp.dll`. See [.NET Core application deployment](../deploying/index.md) for to learn about deployment options.
-
-## Options
-
-# [.NET Core 2.1](#tab/netcore21)
-
-`--additional-deps <PATH>`
-
-Path to an additional *.deps.json* file.
-
-`--additionalprobingpath <PATH>`
-
-Path containing probing policy and assemblies to probe.
-
-`--depsfile`
-
-Path to a *deps.json* file.
-
-A *deps.json* file contains a list of dependencies, compilation dependencies, and version information used to address assembly conflicts. For more information about this file, see [Runtime Configuration Files](https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md) on GitHub.
-
-`-d|--diagnostics`
-
-Enables diagnostic output.
-
-`--fx-version <VERSION>`
-
-Version of the .NET Core runtime to use to run the application.
-
-`-h|--help`
-
-Prints out documentation for a given command, such as `dotnet build --help`. `dotnet --help` prints a list of available commands.
-
-`--info`
-
-Prints out detailed information about a .NET Core installation and the machine environment, such as the current operating system, and commit SHA of the .NET Core version.
-
-`--list-runtimes`
-
-Displays the installed .NET Core runtimes.
-
-`--list-sdks`
-
-Displays the installed .NET Core SDKs.
-
-`--roll-forward-on-no-candidate-fx <N>`
-
-Defines behavior when the required shared framework is not available. `N` can be:
-
-- `0` - Disable even minor version roll forward.
-- `1` - Roll forward on minor version, but not on major version. This is the default behavior.
-- `2` - Roll forward on minor and major versions.
-
- For more information, see [Roll forward](../whats-new/dotnet-core-2-1.md#roll-forward).
-
-`--runtimeconfig`
+To run an application (.NET Core SDK 3.x):
 
-Path to a *runtimeconfig.json* file.
-
-A *runtimeconfig.json* file is a configuration file containing run-time settings. For more information, see [.NET Core run-time configuration settings](../run-time-config/index.md#runtimeconfigjson).
+```dotnetcli
+dotnet [--additionalprobingpath] [--deps-file] [--additional-deps]
+    [--fx-version]  [--roll-forward]
+    [path-to-application] [arguments]
+```
 
-`-v|--verbosity <LEVEL>`
+## Description
 
-Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. Not supported in every command; see specific command page to determine if this option is available.
+The `dotnet` command has two functions:
 
-`--version`
+- It manages .NET Core source code and binaries.
```

Comment:
I would not say "manage" that sounds like `dotnet` is the only way to work with the source code and/or binaries. For example VS/VSCode will handle the source code as well... and running the app is possible with the generated executable instead of `dotnet` (in fact it's the preferred way for 3.0 and above).

Reply:
Maybe something like "It provides command to work with .NET Core projects"....